src/main/java/slimeknights/mantle/client/screen/book/BookScreen.java

```diff
@@ -690,6 +697,40 @@ private void buildPages() {
     }
   }
 
+  private @Nullable PageData getLeftPage() {
```

Comment:
Minor, but I put `@Nullable` above the method instead of inside. Keep consistent with the style for the project.